src/test/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAgeTest.java

```diff
@@ -33,7 +50,7 @@ public class MaxAgeTest {
 
     @Before
     public void setup() throws Exception {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
You should save the result of this operation in a variable, so that you can close it after the tests are finished (inside `@After`).